webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/snippet/configuration/DirectiveManagement.scala

```diff
@@ -69,13 +69,24 @@ import net.liftweb.http.js.*
 import net.liftweb.http.js.JE.*
 import net.liftweb.http.js.JE.JsArray
 import net.liftweb.http.js.JsCmds.*
-import net.liftweb.json
 import net.liftweb.util.Helpers.*
 import net.liftweb.util.Helpers.TimeSpan
+import org.apache.commons.text.StringEscapeUtils
 import org.joda.time.DateTime
 import scala.xml.*
+import zio.json.*
 
-final case class JsonDirectiveRId(directiveId: String, rev: Option[String])
+/*
+ * This class is only used to parse JS parameter, so the built-in escape
```

Comment:
escaping should not be based on input format but output format, I'd prefer to do it in `updateDirectiveForm`